lib/index.ts

```diff
@@ -18,7 +18,6 @@ export { default as FreshheadsLoadReferencedFilesAdapter } from './adapter/fresh
 export { default as FreshheadsCssAdapter } from './adapter/freshheads/CssAdapter';
 export { default as FreshheadsJavascriptAdapter } from './adapter/freshheads/JavascriptAdapter';
 export { default as FreshheadsTypescriptAdapter } from './adapter/freshheads/TypescriptAdapter';
-export { default as FreshheadsCleanBuildDirectoryAdapter } from './adapter/freshheads/CleanBuildDirectoryAdapter';
```

Comment:
Wel breaking change zo. Je zou er ook voor kunnen kiezen de `CleanBuildDirectoryAdapter` intact te houden, en deze de `clean` variabele op `true` te laten zetten voor `output`?

Reply:
Het komt sowieso in een release met meer breaking changes. En in ons geval is het ook onderdeel van DefaultStackAdapter dus zal je het vaak niet merken.

Voordeel van aparte adapter kan wel zijn dat we die opties eventueel makkelijker kunnen meegeven. Nadeel vind ik dat je best wat code gaan creëren voor op dit moment 1 boolean waarvan wij zeggen dat het best practice is om die gewoon altijd op true te hebben staan.